<launch>
	<!-- <arg name="image_topic" default="/kitti/camera_color_left/image_raw" />

	<arg name="mmdet_pub_rate" default="50" />
	<arg name="is_service" default="false" />
	<arg name="visualization" default="true" /> -->

	<arg name="sensor_config_file" default="$(find sv_slam)/config/sv_slam_config.yaml" />

	<node pkg= "sv_slam" name="sv_slam_node" type="sv_slam_node" output="screen">
		<param name="sensor_config_file" value="$(arg sensor_config_file)" />
	</node>
</launch>
